sklearn/datasets/descr/wine_data.rst

```diff
@@ -0,0 +1,85 @@
+Wine Data Database
```

Comment:
A lot of this is copied verbatim without licence.

Note that this does not quite conform to their citation policy either: https://archive.ics.uci.edu/ml/citation_policy.html